package exceptions

class ReachedLastIntegerException : Exception()
